diagnosis/getkuberneteslogs.sh

```diff
@@ -130,17 +130,18 @@ printUsage()
     echo "  -u, --user                        The administrator username for the cluster VMs"
     echo "  -i, --identity-file               RSA private key tied to the public key used to create the Kubernetes cluster (usually named 'id_rsa')"
     echo "  -g, --resource-group              Kubernetes cluster resource group"
+    echo "  -a, --api-model                   AKS Engine Kubernetes cluster definition json file"
```

Comment:
I would not include the `-a` flag, in general people gives it a different meanting, I think `--api-model` is sufficient